### PR TITLE
*: remove redundant code for prewrite and commit.

### PR DESCRIPTION
**Pull request by HuSharp**
Signed-off-by: husharp <jinhao.hu@pingcap.com>

### What is changed and how it works?

Issue Number: Ref #13303 

What's Changed:

```commit-message
remove redundant code for prewrite and commit.
```

### Check List 

Tests 

- Unit test

### Release note 

```release-note
None
```

### CODE REVIEW

File: components/test_raftstore/src/util.rs [hunk 1]
```diff
@@ -818,10 +818,15 @@ pub fn must_kv_read_equal(client: &TikvClient, ctx: Context, key: Vec<u8>, val:
     assert_eq!(get_resp.take_value(), val);
 }
 
-// TODO: replace the redundant code
-pub fn complete_data_commit(client: &TikvClient, ctx: &Context, ts: u64, k: Vec<u8>, v: Vec<u8>) {
+pub fn complete_data_commit(
```
Comment (JmPotato) [inline components/test_raftstore/src/util.rs:821]: I suggest using a more direct name.

```suggestion
pub fn write_and_read_key(
```

**Review by JmPotato — APPROVED** [review]

**Review by JmPotato — APPROVED** [review]

**Comment by HuSharp** [comment]
@Connor1996 Plz take a look, thx!!!

**Review by Connor1996 — APPROVED** [review]
LGTM

**Pull request merged**